core/templating/templating.go

```diff
@@ -79,7 +78,24 @@ func NewEnrichedTemplator(journal *journal.Journal) *Templator {
 	helperMethodMap["randomUuid"] = t.randomUuid
 	helperMethodMap["replace"] = t.replace
 	helperMethodMap["split"] = t.split
-	helperMethodMap["concat"] = t.concat
+	//helperMethodMap["concat"] = t.concat
+
+	// Register concatMany to accept a slice of interface{}
+	raymond.RemoveHelper("concat")
+	raymond.RegisterHelper("concat", func(args ...interface{}) string {
+		var parts []string
+		for _, arg := range args {
+			// If arg is a slice, flatten it
+			if s, ok := arg.([]interface{}); ok {
+				for _, v := range s {
+					parts = append(parts, fmt.Sprint(v))
+				}
+			} else {
+				parts = append(parts, fmt.Sprint(arg))
+			}
+		}
+		return strings.Join(parts, "")
+	})
```

Comment:
do we need this? you have already defined a `concat` function: https://github.com/SpectoLabs/hoverfly/pull/1201/commits/bbd9d3295f6169701022dfb53a9854eee2293bae#diff-794c9f946ba3753b6d7e501b54401cd2a6029c65ea921f46104e3b3b29c6102fR120